src/appsettings.json

```diff
@@ -1 +1 @@
-a4ceb0babd1119d50b8414a2285ccfa174a583dae3bb0d7dfbc3c50cbe7d0ba29b84847a1ee7bf1e13a53df080a43deb1d010c97a5e57e1b3d27bd4eae4c96fa3bb0644e27afc5b3a713c05bad69ed37e508f8c1b83711c4f5ecc1e63c7bddff2a2748f7ac9e20782755fd1a321b72e68608bd8ac4b8cb
\ No newline at end of file
+034be0c00fb358e3252d55aee5ac0eaf97876234093bea6b7743730556a41c1ae62992812f3ab32e365bfa46b0b8162e63d2a4ffdfdcaae54a864d81ba8e92c7469ea299ab330fa31eb4ecc39890ed385cb53e551fe786450a24cf12ff2d1b366ff4c396dc5f6874b32bb8ebd9dae03ae1bb8f46963b15
```

Comment:
Why is this updated? And why is this saved in a `.json` file?

Reply:
This file is encrypted before pushed to remote branches. The content of the file changed as I changed the test encryption key to the official one. I kept the file extension because the decrypted content would be in that format. And so it becomes more clear on application level what you can expect.

Reply:
ah okay, didn't realize the file was encrypted.